sound/soc/sof/loader.c

```diff
@@ -130,6 +130,18 @@ int snd_sof_fw_parse_ext_data(struct snd_sof_dev *sdev, u32 bar, u32 offset)
 }
 EXPORT_SYMBOL(snd_sof_fw_parse_ext_data);
 
+static int ext_man_get_fw_version(struct snd_sof_dev *sdev,
+				  const struct ext_man_elem_header *hdr)
+{
+	const struct ext_man_fw_version *v;
+
+	v = container_of(hdr, struct ext_man_fw_version, hdr);
+
+	memcpy(&sdev->fw_ready, &v->version, sizeof(v->version));
```

Comment:
@ktrzcinx To be correct, shouldn't you be copying the data to "version" field of "fw_ready"..?

Reply:
good catch, changed